versioned_docs/version-8.3/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -29,7 +29,6 @@ The officially offered [client libraries](../../../apis-tools/working-with-apis-
 Currently, Camunda 8 SaaS supports the following scopes:
 
 - Zeebe - Access to the [Zeebe gRPC](/apis-tools/grpc.md) API.
-- Tasklist - Access to the [Tasklist GraphQL](/apis-tools/tasklist-api/tasklist-api-overview.md) API.
```

Comment:
As we still support Tasklist V1, we should add here the reference to Tasklist API (https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/tasklist-api-rest-overview/)